src/Parser/Paladin/Retribution/Modules/Talents/Inquisition.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+// This module looks at the relative amount of damage buffed rather than strict uptime to be more accurate for fights with high general downtime
+
+class Inquisition extends Analyzer {
+  buffedDamage = 0;
+
+  constructor(...args) {
+    super(...args);
+    this.active = this.selectedCombatant.hasTalent(SPELLS.INQUISITION_TALENT.id);
+  }
+
+  on_byPlayer_damage(event) {
+    if (this.selectedCombatant.hasBuff(SPELLS.INQUISITION_TALENT.id)) {
+      this.buffedDamage += event.amount + (event.absorbed || 0);
+    }
+  }
+  get efficiency() {
+    return this.owner.getPercentageOfTotalDamageDone(this.buffedDamage);
+  }
+
+  get suggestionThresholds() {
+    return {
+      actual: this.efficiency,
+      isLessThan: {
+        minor: 0.95,
+        average: 0.9,
+        major: 0.85,
+      },
+      style: 'percentage',
+    };
+  }
+
+  suggestions(when) {
+    when(this.suggestionThresholds).addSuggestion((suggest, actual, recommended) => {
+      return suggest(<React.Fragment>Your <SpellLink id={SPELLS.INQUISITION_TALENT.id} icon /> efficiency is low. You should aim to have it active as often as possible while dealing damage</React.Fragment>)
+        .icon(SPELLS.INQUISITION_TALENT.icon)
+        .actual(`${formatPercentage(this.efficiency)}% of damage buffed`)
+        .recommended(`>${formatPercentage(recommended)}% is recommended`);
+    });
+  }
+
+  statistic() {
+    return (
+      <StatisticBox
+        icon={<SpellIcon id={SPELLS.INQUISITION_TALENT.id} />}
+        value={`${formatPercentage(this.efficiency)}%`}
+        label="Buffed Damage"
```

Comment:
Just a nitpicky thing, but I think it would be more correct to word it "Damage done while buffed" instead (similar to how the tooltip words it). I would usually understand "Buffed Damage" as the percent of base damage buffed (not including the damage from the buff itself).